src/components/DeleteCars/DeleteList.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import EachCarDelete from './EachCarDelete';
+
+const DeleteList = () => {
+  const cars = useSelector((state) => state.cars.cars);
```

Comment:
- [ ] When I delete a car and then go to the dashboard, the car still shows up:

![image](https://github.com/tan12082001/Vehicle-Booking-App-Frontend/assets/57421823/d6cfbfc1-5e69-4417-a494-02576abd5405)

![image](https://github.com/tan12082001/Vehicle-Booking-App-Frontend/assets/57421823/3926e92f-db4e-4a7c-ad89-dd9ba7642e93)

This is because the cars list is not updated unless the car is reloaded.

Kindly, make it so when a car is deleted, a new request is made to the server to request and update the cars list so the car does not show up anymore when deleting it,  without needing to reload the page. 👍